core/src/main/java/org/ldaptive/AbstractConnectionValidator.java

```diff
@@ -94,7 +119,10 @@ public Supplier<Boolean> applyAsync(final Connection conn)
           // waits indefinitely for the validation response
           latch.await();
         } else {
-          latch.await(getValidateTimeout().toMillis(), TimeUnit.MILLISECONDS);
```

Comment:
Since the code ignores the return value, the previous behavior is unclear. Can you please explain what would happen in that case with the original code?

Reply:
You're waiting on the function passed to #applyAsync to give you a boolean. It's up to that implementation to determine when that occurs, but generally it's after an LDAP operation has occurred. So the original version simply used the default value of the `AtomicBoolean` (which is false) when it is done waiting on the `latch`.

Reply:
I see, thanks.